ui/src/app/components/rollout/rollout.tsx

```diff
@@ -297,8 +297,23 @@ const Step = (props: {step: GithubComArgoprojArgoRolloutsPkgApisRolloutsV1alpha1
     return (
         <React.Fragment>
             <EffectDiv className={`steps__step ${props.complete ? 'steps__step--complete' : ''} ${props.current ? 'steps__step--current' : ''}`}>
-                <i className={`fa ${icon}`} /> {content}
-                {unit}
+                <EffectDiv className={`steps__step-title ${props.step.experiment ? 'steps__step-title--experiment' : ''}`}>
+                    <i className={`fa ${icon}`} /> {content}
+                    {unit}
+                </EffectDiv>
+                {props.step.experiment?.templates && (
+                    <EffectDiv className='steps__step__content'>
+                        {props.step.experiment?.templates.map((template) => {
+                            return (
+                                <EffectDiv className='steps__step__content-body'>
+                                    <EffectDiv>name: {template.name}</EffectDiv>
```

Comment:
Could you place the labels and values on separate lines to better visually separate them? Maybe we could capitalize the labels as well, something like this: 

<img width="417" alt="Screen Shot 2022-02-15 at 16 07 27" src="https://user-images.githubusercontent.com/6250584/154171008-02d0395f-7b4c-469c-b290-1a573cdd0d13.png">

Reply:
Sure, will modify.

Reply:
@rbreeze Each template consists of name, specRef and weight. An experiment can consist of many templates. How do you want to segregate the templates.